infrastructure/terraform/README.md

```diff
@@ -0,0 +1,29 @@
+# OpenTofu based deployment
+
+## Requirements
+
+- OpenTofu
+- GCloud CLI
+- GCP Project
+
+## Bootstrap
+
+To bootstrap the account
+
+## User access
+
+To allow a user to execute the GKE terraform, add it to the [user-access module](./user-access/README.md).
```

Comment:
This link is returning 404 I guess. Can you please doublecheck that this is a valid link?